napari/utils/colormaps/_tests/test_colormap.py

```diff
@@ -101,3 +101,11 @@ def test_colormap_equality():
 def test_colormap_recreate():
     c_map = Colormap("black")
     Colormap(**c_map.dict())
+
+
+def test_mapped_shape():
+    for ndim in range(1, 5):
```

Comment:
Optional: I'd prefer to see this use `pytest.mark.parametrize` so that I can see exactly which dimensions fail/pass, rather than fail at the first.